File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseUsuario.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.UsuarioDTO;

/**
 * Objeto de respuesta con la informacion del usuario
 * 
 * @author Flor
 *
 */
public class ServerResponseUsuario {

	private String idUsuario = null;

	private List<UsuarioDTO> listaUsuario = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseUsuario() {
		super();
	}

	public String getIdUsuario() {
		return idUsuario;
	}

	public void setIdUsuario(String idUsuario) {
		this.idUsuario = idUsuario;
	}

	public List<UsuarioDTO> getListaUsuario() {
		return listaUsuario;
	}

	public void setListaUsuario(List<UsuarioDTO> listaUsuario) {
		this.listaUsuario = listaUsuario;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseUsuario.cs
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseUsuario
    {
        private string _idUsuario;

        public string idUsuario
        {
            get { return _idUsuario; }
            set { _idUsuario = value; }
        }

        private List<UsuarioDTO> _listaUsuario;

        public List<UsuarioDTO> listaUsuario
        {
            get { return _listaUsuario; }
            set { _listaUsuario = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
